assets/i18n/id.json

```diff
@@ -25,245 +24,238 @@
+    "dgc-twitter": "Twitter",
+    "dgc-getting-started": "Bagaimana cara memulai menggunakan Dogecoin", 
+    "dgc-setup-wallet": "Membuat sebuah dompet Dogecoin itu <strong>gampang banget</strong>, asli! Periksa panduan memulainya dibawah.", 
+    "dgc-edu-help": "Kalau kamu butuh bantuan, tanyakan di Reddit pada r/dogeducation! <br /> Kita adalah sekelompok orang yang akan membantu dan menjawab setiap pertanyaan yang Kamu ajukan!",
+    "dgc-footer-waiting": "<strong>Jadi tunggu apa lagi?</strong> Mulai pakai Dogecoin hari ini!", 
+    "dgc-footer-getting-started": "Memulai.", 
+    "dgc-footer-learn-how": "Pelajari bagaimana cara menggunakan Dogecoin.", 
     "dgc-community-title": "Komunitas.", 
-    "dgc-community-participate": "Berpartisipasi dalam komunitas.", 
+    "dgc-community-participate": "Berpartisipasi dalam Komunitas.", 
     "dgc-get-dogecoin-title": "Dapatkan Dogecoin.", 
-    "dgc-get-dogecoin-how": "Temukan Cara mendapatkan Dogecoin.",
+    "dgc-get-dogecoin-how": "Cari tau bagaimana cara mendapatkan Dogecoin.",
     "dgc-stores-title": "Toko.", 
-    "dgc-stores-buy": "Beli Dogecoin.",
-    "dgc-footer-credits": "Dibuat oleh <a href=\"https://ummjackson.com\" target=\"_blank\">Jackson Palmer</a> & <a href=\"https://twitter.com/BillyM2k\" target=\"_blank\">Shibetoshi Nakamoto</a>. Desain Website oleh <a href=\"https://www.reddit.com/user/edswf1\" target=\"_blank\">/u/edswf1</a>. Logo karya <a href=\"http://christinemix.com\" target=\"_blank\">Christine Ricks</a>.",
+    "dgc-stores-buy": "Beli dengan Dogecoin.",
+    "dgc-footer-credits": "Dibuat Oleh <a href=\"https://ummjackson.com\" target=\"_blank\">Jackson Palmer</a> & <a href=\"https://twitter.com/BillyM2k\" target=\"_blank\">Shibetoshi Nakamoto</a>. Situs Web dibuat oleh <a href=\"https://www.reddit.com/user/edswf1\" target=\"_blank\">/u/edswf1</a>. Logo dibuat oleh <a href=\"http://christinemix.com\" target=\"_blank\">Christine Ricks</a>.",
     "dgc-footer-copyright": "© The Ðogecoin Project", 
-    "dgc-browser-modal-title": "PILIH SEBUAH DOMPET ONLINE:", 
-    "dgc-browser-modal-online": "Dompet online adalah cara tercepat dan temudah untuk menggunakan Dogecoin, tetapi tidak memiliki keamanan untuk menyimpan dompet Anda di komputer lokalmu. <br /><br /> Kita menyarankan kamu untuk hanya menyimpan sedikit Dogecoin di dompet online kapan saja. </br /> Terdapat beberapa dompet berbasis browser yang dapat dipilih:",
-    "dgc-browser-modal-dogechain": "Dompet DogeChain", 
+    "dgc-browser-modal-title": "PILIH SEBUAH DOMPET DIGITAL:", 
+    "dgc-browser-modal-online": "Dompet digital adalah cara tercepat dan termudah untuk menggunakan Dogecoin, tapi kurang aman jika disimpan pada komputer lokal mu. <br /><br />Kami merekomendasikanmu menyimpan Dogecoin dalam jumlah sedikit saja jika menggunakan dompet digital.  <br />Terdapat beberapa dompet berbasis peramban yang tersedia untuk kamu pakai:",
+    "dgc-browser-modal-dogechain": "Domet DogeChain", 
```

Comment:
It should be Dompet instead of Domet right? 

https://kbbi.kemdikbud.go.id/entri/Dompet